src/utils/mesh_client.ts

```diff
@@ -0,0 +1,47 @@
+import { SignedOrder, ValidationResults, WSClient, WSOpts } from '@0x/mesh-rpc-client';
+import Axios from 'axios';
+import * as _ from 'lodash';
+
+import { MESH_ORDERS_BATCH_SIZE } from '../constants';
+
+import { utils } from './utils';
+
+export class MeshClient extends WSClient {
+    public async addOrdersViaHttpAsync(orders: SignedOrder[], pinned: boolean = false): Promise<ValidationResults> {
+        if (_.isEmpty(this.httpURI)) {
+            return super.addOrdersAsync(orders, pinned);
+        } else {
```

Comment:
It should just fail right? The function name here is misleading if there is a case where it would post over websockets. You could also rename the function.

Reply:
renaming function